scripts/utils/ntc_thermistor_table.py

```diff
@@ -46,24 +46,15 @@ def main(
 ) -> None:
     temps_range = range(temp_init, temp_final + interval - 1, interval)
 
-    print(f"/* NTC Thermistor Table Generated with {os.path.basename(__file__)} */\n")
-    print(f"/ {{")
-    print(
-        f"\tthermistor_R25_{int(r25)}_B_{int(beta)}: thermistor-R25-{int(r25)}-B-{int(beta)} {{"
```

Comment:
Lets keep this portion.  The idea is to create a DT node that one can copy/paste into their device tree file.  We can drop the `status` and `compatible` like you've already done.

Reply:
Done